plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn2/project-with-lockfile/.yarnrc.yml

```diff
@@ -1,3 +1 @@
-nodeLinker: node-modules
-
 yarnPath: .yarn/releases/yarn-3.2.1.cjs
```

Comment:
interresting, I hadn't noticed existance of all these entries.
When writing [1] I thought this was introduce only in version 4.

[1] https://github.com/oss-review-toolkit/ort/pull/10990/commits/cb619d050c068d20cc0c9fbfd892f6ef38d2b70e#diff-983686ff1ae9f49884f63a6d89e79f5aca8ae8089da3750f2b065665326b7530